src/main/resources/org/jenkinsci/plugins/scriptsecurity/scripts/ScriptApproval/tabs/classPath_pending.jelly

```diff
@@ -0,0 +1,74 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler">
+    <div class="pane-frame">
+        <p id="pendingClasspathEntries-none">
+            No pending classpath entry approvals.
+        </p>
+        <div id="pendingClasspathEntries">
+            Classpath entries pending approval. (Beware of remote URLs, workspace files, or anything else that might change without your notice.)
+        </div>
+    </div>
+    
+    <script>
+        var mgr = <st:bind value="${it}"/>;
+        function renderPendingClasspathEntries(pendingClasspathEntries) {
+            if (pendingClasspathEntries.length == 0) {
```

Comment:
Is this new inline JS and prototype code? I would suggest extracting this and use vanilla JS

Reply:
It's part of the copy-paste/transfer from the existing script, nothing changed there (yet).